app/src/main/java/deakin/gopher/guardian/services/UserDataService.kt

```diff
@@ -27,14 +27,4 @@ class UserDataService : FireStoreDataService() {
                 Log.e(dataServiceName, "Failed to create user $userId: $e")
             }
     }
-    // Amending UserDataService to include last password change timestamp
-    fun updateLastPasswordChangeTimestamp(userId: String, timestamp: Long) {
-        usersCollection.document(userId).update("lastPasswordChange", timestamp)
-            .addOnSuccessListener {
-                Log.v(dataServiceName, "Updated lastPasswordChange for User $userId")
-            }
-            .addOnFailureListener { e ->
-                Log.e(dataServiceName, "Failed to update lastPasswordChange for user $userId: $e")
-            }
-    }
```

Comment:
Same question here; I was under the impression that this was a feature request. Why are we removing it now?

Reply:
These will be added later with correct logic. Currently these were causing the app not able to build on who ever forks from this repo.